### ISSUE TITLE
Tuples in `sys.config` handled erroneously, breaks :ip

### ISSUE THREAD

**Issue by martin-langhoff** [bitwalker/conform#73]
Tested with conform `2.0`. If the application's `sys.config` contains a tuple:

```
[{cqerl,[{maps,true},
         {cassandra_nodes,[{<<"127.0.0.1">>,<<"9042">>}]},...
```

and the schema contains:

```
    "cqerl.cassandra_nodes": [
      datatype: [ list: :ip ],
      default: [ {"127.0.0.1", "9099"} ], # note canary in port value
      to: "cqerl.cassandra_nodes"
     ],
```

conform will generate an erroneous output `sys.config` with a list of lists (instead of a list of tuples) holding the data from the input `sys.config` list of tuples:

```
{cqerl,[{cassandra_nodes,[[<<"127.0.0.1">>,<<"9042">>]]},
```

**Comment by jeffweiss** [comment]
I think this might be related to https://github.com/bitwalker/conform/blob/master/lib/conform/utils/utils.ex#L83

**Comment by bitwalker** [comment]
Now that #74 is merged, is this issue addressed?

**Comment by bitwalker** [comment]
@jeffweiss ping :)

**Comment by jeffweiss** [comment]
@bitwalker this is solved for **me**, but I'm not using `:ip`. I was using a more generic list of tuples of binary and list.
```elixir
[
  {"config element 1", []},
  {"config element 2", []},
]
```

I *presume* that since my case is fixed that `:ip` is *also* fixed.

**Comment by jeffweiss** [comment]
Oops, that comment was in draft while I wrote some tests to validate the fix. @bitwalker yes, it's fixed. #78 has tests that fail w/ 2.0.0 and pass with master. 🎉

**Comment by bitwalker** [comment]
Ok excellent, I'll get a new release pushed to hex today

**Comment by spscream** [comment]
@bitwalker it's still not fixed in the latest release :( Did you push it on hex already?

**Comment by bitwalker** [comment]
@spscream Yes, I released 2.1.0 tonight.
